pyaedt/edb_core/siwave.py

```diff
@@ -740,8 +740,24 @@ def create_dc_terminal(
         return self.create_pin_group_terminal(dc_source)
 
     @pyaedt_function_handler()
-    def create_exec_file(self, add_dc=False, add_ac=False, add_syz=False):
-        """Create an executable file."""
+    def create_exec_file(
+        self, add_dc=False, add_ac=False, add_syz=False, export_touchstone=False, touchstone_file_path=""
+    ):
+        """Create an executable file.
+        Parameters
+        ----------
+        add_dc : bool, optional
+            Add the DC option in the exec file
+        add_ac : bool, optional
+            Add the AC option in the exec file.
+        add_syz : bool, optional
+            Add the SYZ option in the exec file
+        export_touchstone : bool, optional
+            Add the Touchstone file export option in the exec file
+        touchstone_file_path : str, optional
+            When string is empty and ``export_touchstone`` == ``True``, the default file path will be at the project
+            location.
```

Comment:
```suggestion
        add_dc : bool, optional
            Whether to add the DC option in the EXE file. The default is ``False``.
        add_ac : bool, optional
            Whether to add the AC option in the EXE file. The default is
            ``False``.
        add_syz : bool, optional
            Whether to add the SYZ option in the EXE file
        export_touchstone : bool, optional
            Add the Touchstone file export option in the EXE file.
            The default is ``False``.
        touchstone_file_path : str, optional
            File path for the Touchstone file. The default is ``""``.  When no path is
            specified and ``export_touchstone=True``, the path for the project is
            used.
```
I don't see `add_syz`` in the method signature.